app/services/scene-collections/nodes/sources.ts

```diff
@@ -218,7 +216,7 @@ export class SourcesNode extends Node<ISchema, {}> {
             name: filter.name,
             type: filter.type,
             settings: filter.settings,
-            enabled: (filter.enabled === void 0) ? true : filter.enabled
```

Comment:
I always thought it is nothing bad to add additional brackets, especially in a ternary operator O_o

Reply:
Yeah I actually ran Prettier on this file and this was one of the changes it made, which had nothing to do with TSLint.  This is actually a great example of Prettier changing something that really didn't need to be changed, and arguably the end result is worse.